packages/fast-tooling/src/web-components/css-box-model/css-box-model.spec.ts

```diff
@@ -0,0 +1,750 @@
+import { expect } from "chai";
+import { DOM, html } from "@microsoft/fast-element";
+import { DesignSystem } from "@microsoft/fast-foundation";
+import { fixture } from "../../__test__/fixture";
+import { CSSBoxModel } from "./css-box-model";
+import { fastToolingCSSBoxModel } from ".";
+
+async function setup() {
+    const { element, connect, disconnect } = await fixture<CSSBoxModel>(
+        html`
+            <fast-tooling-css-box-model id="css-box-model"></fast-tooling-css-box-model>
+        `,
+        {
+            designSystem: DesignSystem.getOrCreate()
+                .withPrefix("fast-tooling")
+                .register(fastToolingCSSBoxModel()),
+        }
+    );
+
+    return { element, connect, disconnect };
+}
+
+describe("CSSBoxModel", () => {
+    it("should initialize and render", async () => {
+        const { element, connect, disconnect } = await setup();
+        await connect();
+
+        const event = new Event("change", {} as Event);
+        let wasChanged: boolean = false;
+
+        expect(element.id).to.equal("css-box-model");
+
+        element.addEventListener("change", (e: Event) => {
+            wasChanged = true;
+        });
+        const marginTopInput = element.shadowRoot.getElementById(
+            "margin-top"
+        ) as HTMLInputElement;
+        marginTopInput.value = "1px";
+        marginTopInput.dispatchEvent(event);
+
+        await DOM.nextUpdate();
+
+        expect(wasChanged).to.equal(true);
+        expect(element.value).to.equal("margin-top:1px;");
+
+        await disconnect();
+    });
+    it("should correctly parse margin values", async () => {
```

Comment:
nit: I would avoid the word "correctly" in the phrasing of the tests as it's redundant
```suggestion
    it("should parse margin values", async () => {
```

Reply:
Removed all instances.